code/online_challenges/src/project_euler/problem_067/README.md

```diff
@@ -0,0 +1,26 @@
+# Project Euler Problem #067: Maximum path sum II
+
+([Problem Link](https://projecteuler.net/problem=18))
+By starting at the top of the triangle below and moving to adjacent numbers on the row below, the maximum total from top to bottom is 23.
+<p align="center">
+
+   `3`
```

Comment:
Use triple backtick style code formatting to preserve indentation.
```
\```
content
\```
```

(Without the slashes)